<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1147</width>
    <height>744</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1147</width>
    <height>744</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1147</width>
    <height>744</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Рисование на виртульных функциях</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="ellips_radioButton">
        <property name="text">
         <string>Эллипс</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="sqsare_radioButton">
        <property name="text">
         <string>Квадрат</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rhomb_radioButton">
        <property name="text">
         <string>Ромб</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_s1">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>X центр элипса</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="x_centr_left"/>
      </item>
      <item>
       <widget class="QLabel" name="label_s2">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Y центр эллипса</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="y_centr_top"/>
      </item>
      <item>
       <widget class="QLabel" name="label_s3">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Вертикальный радиус</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="v_r_lenght"/>
      </item>
      <item>
       <widget class="QLabel" name="label_s4">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Горизонтальный радиус</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="h_r_height">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Угол поворота(по часовой стрелке)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="angle_spinBox">
        <property name="maximum">
         <number>359</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="add_ellips">
        <property name="text">
         <string>Добавить фигуру</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="del_last">
        <property name="text">
         <string>Удалить последнюю фигуру</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Новые координаты</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QSpinBox" name="new_x"/>
        </item>
        <item>
         <widget class="QSpinBox" name="new_y"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="move_button">
        <property name="text">
         <string>Переместить последнюю фигуру</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Масштаб</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="scale_spinBox">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1147</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
